doc/howto/network_ovn_setup.md

```diff
@@ -1,33 +1,155 @@
+   | NAME |  STATE  |        IPV4         |                     IPV6                     |   TYPE    | SNAPSHOTS |
+   +------+---------+---------------------+----------------------------------------------+-----------+-----------+
+   | c1   | RUNNING | 192.0.2.2 (eth0)    | 2001:db8:cff3:5089:216:3eff:fef0:549f (eth0) | CONTAINER | 0         |
+   +------+---------+---------------------+----------------------------------------------+-----------+-----------+
+   ```
+
+## Set up a LXD cluster on OVN
+
 ```{youtube} https://www.youtube.com/watch?v=1M__Rm9iZb8
 ```
 
-This will create a standalone OVN network that is connected to the parent network lxdbr0 for outbound connectivity.
+Complete the following steps to set up a LXD cluster that uses an OVN network.
 
-Install the OVN tools and configure the OVN integration bridge on the local server:
+Just like LXD, OVN's distributed database must be run on a cluster that consists of an odd number of members.
+The following instructions use the minimum of three servers, which run both the distributed database for OVN and the OVN controller.
+In addition, you can add any number of servers to the LXD cluster that run only the OVN controller.
+See the linked YouTube video for the complete tutorial using four machines.
 
-```
-sudo apt install ovn-host ovn-central
-sudo ovs-vsctl set open_vswitch . \
-  external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
-  external_ids:ovn-encap-type=geneve \
-  external_ids:ovn-encap-ip=127.0.0.1
-```
+1. Complete the following steps on the three machines that you want to run OVN's distributed database:
 
-Create an OVN network and an instance using it:
+   a. Install the OVN tools:
 
-```
-lxc network set lxdbr0 ipv4.dhcp.ranges=... ipv4.ovn.ranges=... # Allocate IP range for OVN gateways.
-lxc network create ovntest --type=ovn network=lxdbr0
-lxc init ubuntu:22.04 c1
-lxc config device override c1 eth0 network=ovntest
-lxc start c1
-lxc ls
-+------+---------+---------------------+----------------------------------------------+-----------+-----------+
-| NAME |  STATE  |        IPV4         |                     IPV6                     |   TYPE    | SNAPSHOTS |
-+------+---------+---------------------+----------------------------------------------+-----------+-----------+
-| c1   | RUNNING | 10.254.118.2 (eth0) | fd42:887:cff3:5089:216:3eff:fef0:549f (eth0) | CONTAINER | 0         |
-+------+---------+---------------------+----------------------------------------------+-----------+-----------+
-```
+        sudo apt install ovn-central ovn-host
+
+   b. Mark the OVN services as enabled to ensure that they are started when the machine boots:
+
+        systemctl enable ovn-central
+        systemctl enable ovn-host
+
+   c. Stop OVN for now:
+
+        systemctl stop ovn-central
+
+   d. Note down the IP address of the machine:
+
+        ip -4 a
+
+1. On the remaining machines, install only `ovn-host` and make sure it is enabled:
```

Comment:
Can you move this section below the 

> On the first three machines, configure OVN:

So that we cover setting up the OVN cluster DB members first and then cover doing the non-DB members separately.
Rather than alternative between the different server types.